frontend/src/pages/notebookController/SpawnerPage.tsx

```diff
@@ -447,7 +483,16 @@ const SpawnerPage: React.FC<SpawnerPageProps> = React.memo(({ setStartModalShown
           </Button>
         </ActionGroup>
       </Form>
-    </>
+      {startShown && (
+        <StartServerModal
+          isNotebookRunning={isNotebookRunning}
+          notebook={notebook}
+          startShown={startShown}
+          setStartModalShown={setStartShown}
+          onClose={onModalClose}
+        />
+      )}
```

Comment:
```suggestion
        <StartServerModal
          isNotebookRunning={isNotebookRunning}
          notebook={notebook}
          startShown={startShown}
          setStartModalShown={setStartShown}
          onClose={onModalClose}
        />
```
Would you not just want this? You're already using the `startShown` as the modal open state... mounting/unmounting the modal kills any animations if I recall correctly.

Reply:
Oh, it's probably some code I forgot to clean up when I move this from another file. I will change this.